jvm/src/test/scala/org/scalacheck/GenSpecification.scala

```diff
@@ -400,7 +400,7 @@ object GenSpecification extends Properties("Gen") {
   )
 
   property("22 field case class works") = forAll(Gen.resultOf(Full22.tupled)){
-    Full22.unapply(_).get.isInstanceOf[Tuple22[_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_]]
+    _.isInstanceOf[Full22]
```

Comment:
This won't compile if ScalaCheck's tuple support is defective, so could probably just be written simply like the rest of the similar tests found earlier in the suite (`resultOf1`, `resultOf2`, ...):

```scala
  property("22 field case class works") =
    forAll(Gen.resultOf(Full22.tupled)) { _ => true }
```